Marlin/Configuration.h

```diff
@@ -292,7 +292,35 @@ your extruder heater takes 2 minutes to hit the target on heating.
 
 
 //===========================================================================
-//============================= Mechanical Settings =========================
+//=============================Pneumatics Settings===========================
+//===========================================================================
+
+// Define this if you are using pneumatic direct write on the RAMBo AND you are not
+// using HEAT_1 output
+#define PNEUMATICS
+
+#ifdef PNEUMATICS
+
+// Set Pressure Sensor Type HERE:
+// -----------------------------
+// 0 - No Pressure Sensor
+// 1 - Kavlico P255-50G-D1A
+// 2 - American Sensor Tech. 4100 Series (1-5V output)
+//
+
+  #define PNEUMATIC_SENSOR 2
+  
+  // 0 is a valid pressure reading
+  #define PNEUMATIC_MIN -1
+  
+  // If the pressure goes above this value the pump will be turned off. This prevents
+  // the tank from being overpressurized.
```

Comment:
It might make sense to leave a comment here that the units on this value are 10ths of PSI